shared/common-adapters/input.native.js

```diff
@@ -66,7 +66,7 @@ class Input extends Component<void, Props, State> {
           onChange={this.props.onChange}
           onChangeText={text => { this.setState({text}); this.props.onChangeText && this.props.onChangeText(text) }} />
         {IOS && <HorizontalLine focused={this.state.inputFocused}/>}
-        {this.props.errorText && <Text type='Error' style={{...errorText, ...this.props.errorStyle}}>{this.props.errorText}</Text>}
+        {!!(this.props.errorText) && <Text type='Error' style={{...errorText, ...this.props.errorStyle}}>{this.props.errorText}</Text>}
```

Comment:
via @MarcoPolo, this fixes `RawText "" must be wrapped in an explicit <Text> component.` when this.props.errorText is blank.